roofit/roofit/inc/RooPyBindFunction.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   Robin Syring, CERN 2024
+ *
+ * Copyright (c) 2023, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#ifndef ROOPYBINDFUNCTION_H
+#define ROOPYBINDFUNCTION_H
+
+#include "RooAbsReal.h"
+#include "RooListProxy.h"
+#include "RooArgList.h"
+
+class RooPyBindFunction : public RooAbsReal {
+public:
+   RooPyBindFunction(const char *name, const char *title, RooArgList &varlist);
+   RooPyBindFunction(const RooPyBindFunction &right, const char *name = nullptr);
+   virtual ~RooPyBindFunction();
+
+   RooPyBindFunction* clone(const char *name) const override;
+   Double_t evaluate() const override;
+   const RooArgList& varlist() const;
+
+protected:
+   RooListProxy m_varlist; // all variables as list of variables
+
+   ClassDefOverride(RooPyBindFunction, 0)
```

Comment:
```suggestion
   ClassDefOverride(RooPyBindFunction, 0);
```
It's not necessary here because it's a preprocessor macro, but to not confuse the automatic code formatting it would be good to end the line with a semicolon too.